dist/release-vespa-rpm.sh

```diff
@@ -19,6 +19,11 @@ readonly CURRENT_BRANCH=$(git branch | grep "^\*" | cut -d' ' -f2)
 git checkout master
 git pull --rebase
 
+# Update the VERSION file on master to be the next releasable version
+echo "$VERSION" | awk -F. '{print $1"."($2+1)".0"}' > VERSION
+git commit -am "Updating VERSION file to next releasable minor version."
+git push
```

Comment:
Should this be in a simple retry a few times loop, to avoid spurious failures when the remote master has changed before git push ?

Reply:
Added a simple retry.